src/frontend/src/components/Banner/Banner.jsx

```diff
@@ -19,51 +19,60 @@ const useStyles = makeStyles(theme => ({
     fontWeight: 'bold',
     opacity: 0.85,
     fontSize: '12rem',
+    display: 'block',
+    top: theme.spacing(20),
+    left: theme.spacing(8),
+    transition: 'linear 250ms all',
```

Comment:
I'm not a huge fan of transitions between sizes.  My vote would be to avoid this.  But overrule me if I'm wrong.